modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/ModelBuilderContext/objectFolderReducer.ts

```diff
@@ -38,10 +38,12 @@ export function ObjectFolderReducer(state: TState, action: TAction): TState {
 				y: 2 * 400,
 			};
 
-			if (nodes.length) {
-				const yPositions = nodes.map((node) => node.position.y);
+			if (objectDefinitionNodes.length) {
+				const yPositions = objectDefinitionNodes.map(
+					(node) => node.position.y
+				);
 				const maximumY = Math.max(...yPositions);
-				const maximumNodesYPosition = nodes.filter(
+				const maximumNodesYPosition = objectDefinitionNodes.filter(
 					(node) => node.position.y === maximumY
```

Comment:
@danvb112 We're following a pattern where the name of the current item is the singular version of the name of the array being iterated. So in this case you should use `objectDefinitionNode` instead of `node`